src/overlays/actors/ovl_En_Ge2/z_en_ge2.c

```diff
@@ -556,7 +556,7 @@ void EnGe2_Update(Actor* thisx, PlayState* play) {
     } else if (this->collider.base.acFlags & AC_HIT) {
         if ((this->collider.info.acHitInfo != NULL) &&
             (this->collider.info.acHitInfo->toucher.dmgFlags & DMG_HOOKSHOT)) {
-            Actor_SetColorFilter(&this->actor, 0, 120, 0, 400);
+            Actor_SetColorFilter(&this->actor, COLORFILTER_COLORFLAG_BLUE, 120, COLORFILTER_BUFFLAG_OPA, 400);
```

Comment:
I took a quick look if every `Actor_SetColorFilter` usage was mindful of the `duration` being 8bits...
Turns out no, at least this one isn't (haven't found others but it was a really quick glance)

`400 = 0b1_1001_0000` so the maxDuration in the bitfield becomes `0b1001_0000 = 144` and there's a bit leaking into the maxIntensity

However the timer is still initialized to 400, I wonder if that causes a noticeable something in-game

Anyway, is this a `@bug` ?

Reply:
I'm not sure if I would count it as a bug, maybe a limitation/bug of the system? It technically allows to set a duration over 255 since that parameter is a `s16` but as you said it is leaking one extra bit, with the weird bitpacking it ends up being an extra 8 (out of 255) of the color intensity. It probably isn't that noticeable but it may be worth checking

Not sure if we should tag it as a `@bug`, or if it should be tagged in the actor or in the `Actor_SetColorFilter` function itself

Reply:
I tested in-game and it doesn't do anything special...

Because `colorFilterTimer` is a u8 so the 400 passed to it also becomes 144 <img src="https://cdn.frankerfacez.com/emoticon/263833/2" height="32" alt="PepeLaugh">

Definitely a bug then imo. The system just doesn't handle durations over 255